test/include/dlaf_test/matrix/util_generic_blas.h

```diff
@@ -61,22 +63,22 @@ auto getSubMatrixMatrixMultiplication(const SizeType a, const SizeType b, const
 
   auto elC = [a, b, k](const GlobalElementIndex ij) {
     if (ij.row() < a || ij.row() > b || ij.col() < a || ij.col() > b)
-      return TypeUtilities<T>::polar(-99, 99);
+      return -TypeUtilities<T>::polar(99, 99);
```

Comment:
Would it be worth adding an assertion to `TypeUtilities::polar` (possibly `DLAF_ASSERT_MODERATE` if it's expected to be called frequently somewhere). https://en.cppreference.com/w/cpp/numeric/complex/polar says "The behavior is undefined if r is negative or NaN, or if theta is infinite."

Reply:
Yes, I'm with you, I thought about that too 😉 It might also reveal other cases under cover.

Reply:
Question: since this is in `DLAF_test` library, which does not depend on `dlaf`, should we just go for plain `assert`?

Reply:
Good point. Yes, plain assert sounds good!

Reply:
Looks good, though I realize only now: why not `DLAF_ASSERT`? I actually thought you were referring to that with plain assert (as opposed to `DLAF_ASSERT_MODERATE` which is not "plain").